docs/architecture/modernize-desktop/deploying-modern-desktop-applications.md

```diff
@@ -0,0 +1,332 @@
+an app because it ends up having some undesired side effects and some already
+installed applications stop working. Moreover, you cannot just restore the
+system to its original state by uninstalling. We are so used to live this
+situation that we have coined terms like "DLL Hell" or "Winrot".
+
+In this chapter, we talk about MSIX the brand-new technology from Microsoft that
+tries to capture the best of previous technologies to provide a solid foundation
+for the packaging technology of the future.
+
+What does a packaging technology have to do with modernization? Well, it turns
+out that packaging is fundamental for the enterprise IT with lots of money
+invested there. Modernization is not only related with using the latest
+technologies but also with reducing time to market between a business
+requirement is defined and the moment your company delivers the feature to your
+client.
+
+## The modern application lifecycle
+
+Today, developers write and build the code for an app and then pass the
+generated assets to the IT pros who reconfigure the app and repackage it
+typically in an MSI or more recent App-V packaging format, which is deployed
+through different channels and tools. One of the main problems with this
+approach is commonly known as "packaging paralysis". The problem is that this
+cycle repeats every time there is an App update or an OS Update.
+
+We can see the process reflected on this picture:
+
+![Modern IT](media\deploying-modern-desktop-applications\modern-it.png)
+
+Companies need a way to break in this packaging cycle three independent cycles:
+
+* OS Updates
+* Application Updates
+* Customization
+
+![IT cycles](media\deploying-modern-desktop-applications\it-cycles.png)
+
+This means we should be able to update the underlying OS without having to
+repackage our apps or enable customizations from IT without the need to
+repackage the original developer package.
+
+This radical change leads us to the new and modern IT lifecycle depicted in the
+following picture:
+
+![Microsoft IT tools](media\deploying-modern-desktop-applications\microsoft-it-tools.png)
+
+Developers create the app and generate an MSIX package that IT pros can consume
+and configure without the need of repackaging. Along with the MSIX technology,
+Microsoft has created tools to allow IT to customize and configure packages
+without repackaging.
+
+## MSIX: The next generation of deployment
+
+Before MSIX, we had several packaging technologies available like setup wizards,
+MSI, ClickOnce, App-V, and scripting. Each of  technologies has their strengths and
+Microsoft has decided to pick the best of all to build MSIX. Therefore, MSIX is
+built on the foundations of these existing technologies picking the best of
+each:
+
+* App-V =\> Containerization
+* ClickOnce =\> Auto updating
+* MSI =\> Easy to distribute
+
+With MSIX, we get one installer technology with all these features.
+
+![MSIX](media\deploying-modern-desktop-applications\msix.png)
```

Comment:
Please review my suggestion here. Not sure if that's what you were trying to show here. You should also make the image name longer.

```suggestion
![Diagram showing the different technologies that had an impact on building MSIX](./media/deploying-modern-desktop-applications/msix.png)
```